### DISCUSSION TITLE
아이디어 불패의 법칙

### DISCUSSION THREAD

**Discussion by hanseulhee** [primus-teoSprint/primus_FE#18] [기록]
# 아이디어 불패의 법칙 스터디

<img width="1141" alt="스크린샷 2024-04-29 오전 12 09 26" src="https://github.com/primus-teoSprint/FE/assets/63100352/e8dfadd4-60a9-466e-ae25-c03ba6a961db">

<img width="647" alt="스크린샷 2024-04-29 오전 12 10 22" src="https://github.com/primus-teoSprint/FE/assets/63100352/aeffcbe2-282e-4b43-8b13-119c48e780b3">

**Comment by chaduhwan** [comment]
아이디어 불패의 법칙 / 알베르토 사보이아

1부 불변의 사실
1장 시장 실패의 법칙
1. 팩트는 확고하고 객관적인 데이터에 기초한다, 견고하고 단호하고 영구적이라는 의미에서 확고하다 => 객관성과 영구성을 지닌다. 38p
    1. 실패할 확률이 가장 높은건 사실이다. 39p
    2. 실패를 객관적이고 분석적인 방식으로 다뤄야한다. 40p
2. 시장 실패와 성공의 정의
    1. 기대에 못미치거나 상반된 것. 42p
    2. 기대에 대한 자신의 기준에 따라 실패와 성공이 가름됨. 43p
3. 실패의 패턴 FLOP
    1. 실패는 출시 운영 전제 때문.55p
        1. 출시 : 신제품의 세일즈, 마케팅, 유통을 위한 노력이 의도한 시장에 영향을 주지 못할 떄
        2. 운영 : 신제품의 디자인, 기능, 안정성이 이용자들의 최저 기대치에도 미달할 떄
        3. 전제 : 사람들이 그냥 아이디어에 관해 관심을 갖지 않을떄
    2. 이 ‘전제’에서 실패하는 경우가 많다 ! 

2장 될놈
1. 될 놈이란 : 유능하게 실행할 경우 시장에서 성공할 신제품 아이디어. 60p
    1. 시장에는 좋은 아이디어와 나쁜아이이디어가 아닌 성공한, 실패한 아이디어만 존재
    2. 아이디어가 성공할지, 우리가 성공할지는 보장 x. 61p
2. 안 될 놈 : 될놈의 반대, 유능하게 실행해도 성공이 안되는 아이디어. 62p
3. 생각 랜드의 의견은 데이터가 아니다. 전문가의 말이라도. 65p
    1. 포커스 그룹의 실패. 71p
        1. 아이디어 전달 문제 : 구체적인 형태가 되기전까지 추상적인 아이디어는 개인의 맥락 내에서 판단된다. 73p
        2. 예측력 문제 : 아이디어를 향후 얼마나 활용할 지 예측하는 것은 거의 불가능. 76p
        3. 적극적 투자가 없다 : 아무런 투자를 하지않는 사람들의 의견은 중요치 않다. 77p
        4. 확즉편향 : 자신의 생각과 상반되는 데이터는 거의 무시한다. 78p
    2. 긍정 오류와 부정 오류.85p
    3. 이를 타파하는 것은 데이터다!

3장 생각은 접어두고 데이터를 모으라
1. 의사과정 결과에 고려되는 데이터의 기준. 93p
    1. 신선함
    2. 확실한 관련성 : 지금 평가하려는 특정 제품이나 의사 결정에 직접적으로 적용 가능한 것. 94p
    3. 알려진 출처 : 어떻게 수집과 필터링 되었는 지 모르는 데이터는 쓰지마라
    4. 통계적 유의성 : 개인적인 경험이나 일회성 이야기는 제시하지 말것. 95p
2. 그들의 데이터 96p
    1. 다른 사람, 프로젝트, 시기, 곳, 방법, 목적 으로 수집하고 편집한 모든 시장 데이터
    2. 5가지의 시나리오
        1. 역사상 처음 낸 아이디어
        2. 다른 사람도 같은 아이디오 혹은 비슷한 아이디어를 생각해냄. 97p
            1. 추진하지 않기로 함
            2. 적극적으로 추진 주이지만 출시하지 않음
            3. 추진하고 출시했지만 실패
            4. 추진하고 출시해서 성공
    3. 위 5 시나리오에서 접근한 데이터에 의존하지 마라. 100p
3. 나만의 데이터를 수집해라. 101p

2부 쓸모있는 데이터를 수집하는 방법
4장 사고 도구
1. 아이디어를 테스트할려면 명확하고 정확하게 언어로 표현할 수 있어야 한다. 107p
2. 시장 호응 가설
    1. 아이디어의 기본 전제와 그에 대한 시장의 호응에 대해 어떤 기대를 하고 있는 지를 짧은 문장으로 표현. 111p
    2. 시장에 대한 정확한 정의를 내리고 해당 시장 수요가 존재한다는 증거를 찾아내는 것. 113p
3. 숫자로 이야기하라. 
    1. 구체적인 검증 가능한 가설로 바꿔 테스트. 115p
4. XYZ 가설
    1. 적어도 X 퍼센트의 Y는 Z할 것이다. 117p
        1. X: 표적 시장의 구체적인 퍼센티지 118p
        2. Y : 표적 시장을 명확하게 설명
        3. Z : 시장이 아디이어에 어떤 식으로 호응할 것인지 예상
    2. 애매 모호함을 제거하는데 효과 120p
    3. 예제 123p
5. 범위 축소 : 지금 당장 실행 가능하고 검증 가능한 가설을 얻는것 => xyz 가설
    1. 표적 시장 축소 Y => y 126p
        1. 대표성은 있으나 한정된 지역에 해당하는 작은 부분집합
        2. 샘플 크기가 너무 작아져서 통계적 유의성을 상실하는 일이 없도록 유의
        3. Ex) 대기질 지수가 100 이상인 전 세계 모든 도시 => 베이징에 있는 토트 아카데미의 학부모 128p

5장 프리토타이핑 도구
1. 프리토타이핑, 프리토타입 137p
    1. IBM 의 음성타이핑기술 예시
    2. 어느 아이디어가 추구하고 만들 가치가 있는지를 값싸고 빠르게 검증하기 위해 설계 139p
    3. 프로토타입, 시제품과 구분 지어야한다. 140p
2. 미캐니컬 테크 프리토타입 145p
    1. 값비싸고 복잡한 기술이나 아직 개발되지 않은 기술을 인간이 은밀히 그 기술을 대신 구현하는 것이 가능할 떄
    2. 폴드포유의 예시 150p
    3. 사람들이 실제 무언가를 투자해야할 때 정말 투자하는 지 155p
3. 피노키오 프리토타입
    1. 팜파일럿 사례 
    2. 모형이나 작동하지 않는 시제품을 실제로 작동하는 척하면서 사용해보는 것
4. 시제품이냐 프리토타입이냐
    1. 시제품의 목적 161p
        1. 이걸 만들 수 있는가?
        2. 의도대로 작동할 것인가?
        3. 얼마나 작게 크게 싸게 에너지 효율적을 만들 수 있을까?
    2. 프리토타입의 목적 162p
        1. 이걸 사용할 것이냐?
        2. 언제 어떻게 얼마나 자주 사용할 것이냐?
        3. 남들이 사줄 것이냐?
        4. 이 제품에 얼마를 지불할려고 할것이냐?
        5. ==> 이걸 만들어야할까?
5. 가짜 문 프리토타입 167p
    1. 어떤 제품이나 서비스가 존재하는 것처럼 보일만한 현관문을 설치하면, 얼마나 많은 사람이 아이디어에 관심을 가질지
    2. 게임 광고 예시 170p
    3. 약간의 기만이 있기에 데이터를 쌓을 수 있도록 도와준 사람들에게 윈윈이 되도록 가치 있는 것을 돌려 주어라
    4. 앤티크 가게 예시 175p
    5. 윤리 문제 179p
6. 외관 프리토타입 185p
    1. 카즈 다이렉트 사례
    2. 가짜 문 프리토타입을 사용하는 것이 비윤리적일 떄 (질변에 대한 치료법 등) 187p
    3. 가짜 문 프리토타입보다 잠재적 비즈니스에 관해 휠씬 더 맣은 것을 알아낼 수 있다.
7. 유튜브 프리토타입 190p
    1. 아직 제대로 개발되지 않았거나 널리 이용 가능하지 않은 제품 아이디어에 생명을 불어넣고 표적시장에서 공유할 수 있게
    2. 시장이 얼마나 아이디어에 관심을 갖는지
    3. 구글 글래스 익스플로러 에디션 사례 191p
        1. 처음의 관심과 투자가 필요하기느 하지만 이것만으로는 해당 제품이 될 놈인지의 여부를 결정할수 없음을 보여줌
        2. 일부 제품과 서비스의 성공에는 반복적인 사용과 지속적인 관심이 중요
    4. 조회수나 좋아요 개수 같은 지표는 데이터가 아님을 기억 195p
    5. 다른 프리토타이핑 기법과 결합하면 더 좋은 결과를 기대
8. 프리토타이핑의 투자 수익률 : 프리토타이핑으로 인해 실제 투자를 하고 실패했을 떄 보다 덜 손해본다. 201p
9. 하룻밤 프리토타입 202p
    1. 장기적 게획이나 투자가 없다.
    2. 프리토타이핑 실험기간이 2시간일 수, 2달일수도 상대적 단기간에 진행한다는 점. 203p
    3. 버진항공 사례
    4. 에어비엔비 사례
    5. 테슬라 팝업 쇼룸 사례 207p
10. 잡입자 프리토타입 209p
    1. 제품을 다른 누군가의 기존 판매 환경에 몰래 끼워 넣는 방법
    2. 평소 비슷한 제품이 놓이는 곳에 제품을 가져다 두고, 사람들이 적극적 위험을 감수하면서 제푸믈 구매하는 지
    3. 월허브 사례 210p
    4. 인터넷에도 가능. 대형회사보단 작은 회사와 접촉해서 협업하는 편이 더 쉽다. 215p
11. 상표 바꾸기 프리토타입
    1. 기존 제품이나 서비스의 외관을 조금만 바꿔서 다른 제품인척 하며 사람들의 관심을 갖는지 보는 것이다.
    2. 하루 지난 초밥 사례
12. 변형과 조합
    1. 라이브 데모 프리토 타입 224p
    2. 한입 프리토 타입 : 아직 완성되지 않은 작품을 공짜로 몇 부분을 보여주며 투자를 유치 받는 방법 226p
        1. 마션 사례
13. 프리토타입의 본질
    1. 적극적 투자가 있는 나만의 데이터를 생성해야 한다.
    2. 빠르게 수행할 수 있어야한다.
    3. 저렴하게 수행할 수 있어야한다. 227p

6장 분석도구
 앞서 사고 도구들이 막현하고 불분명한 아이더를 분명하고 객관적이며 검증 가능한 가설로 바꿔주는 모습을 보였다.
이제 분석도구를 이용해 수집한 데이터를 이해하는 방법을 배워보자. 데이터가 의사결정으로 바뀌는 중요한 순간이다 229p
1. 적극적 투자지표 
    1. 적극적 투자란 무언가에 대한 위험 부담을 진다는 뜻 => 아이디어에 대한 심사숙고 232p
    2. 적극적 투자지표는 표적 시장의 반응 유형에 따라 적극적 투자의 점수를 할당 233p
    3. 토르텔로매틱 사례 234p
    4. 두팀 사례 239p
2. 될놈척도
    1. 적극적 투자가 포함된 나만의 데이터를 수집해서 시장 호응 가설을 확인하는 것만으로 충분치 않다. 241p
    2. 데이터에서 가치를 추출하고 해석, 측정, 비교하기위해 다른 유관과 결합하는 방법이 필요
    3. 될놈 척도 : 만약 시장 호응 가설이 참이면, 이 프리토타이핑 실험이 이런 데이터를 내놓을 확률은 ? 247p
    4. 하루 지난 초밥 사례 
3. 어느 정도 데이터가 필요할까
    1. 결과에 대한 확신을 가지려면 여러 번의 프리토타이핑 실험으로 다수의 xyz가설의 유효성을 확인해야함. 253p
        1. 아이디어에 얼마나 많이 투자할 계획 ?
        2. 잘못될 경우 읽어도 되는 시간과 돈은 어느정도 ?
        3. 의사결정을 내리기 전에 어느 정도의 확실성이 필요?
        4. 지금까지의 실험 결과들이 확실한가, 불확실한가?

3부 유연한 전략
7장 전략도구
1. 생각은 글로벌하게, 테스트는 로컬하게
    1. 야심찬 해외 계획의 수립과 실행에 시간을 쓰기 전에 휠씬 작고 접근하기 쉬운 표적 시장의 하위 시장에서 아이디어를 검증해야 한다는것 267p
    2. 물리적 거리 지리적 지역에만 적용 x, 시장이 개념적으로 여러 집단으로 나뉘거나 업계의 표준이 여럿이라서 각각에 접근하려면 추가적인 투자가 필요할 때에도 268p
    3. 데이터까지의 거리
        1. 테스트는 로컬하게를 잘 따르고 있나 ? => 데이터까지의 거리를 계산해라! 272p
        2. 온라인에서는 ? 디지털단계의 숫자를 세면된다
2. 내일보다는 오늘 테스트하는 게 낫다
    1. 생각랜드에 너무 오래 갇혀있지 마라. 276p
    2. 데이터까지의 시간
        1. 짧을 수록 더 좋다.
3. 싸게, 더 싸게, 제일 싸게 생각하라
    1. 창의성은 제약을 좋아한다. 더 비용을 줄일 수 없는지 생각해라 281p
    2. 데이터까지의 비용
        1. 다양한 화페단위가 들어갈 수 있다. 대접하는 도넛까지도.
4. 고치고 뒤집고 다 해보고 그만둬라
    1. 첫 실험에서 실망스러운 ‘나만의 데이터’가 나왔다고 해서 섣불리 낙담하지 말것. 282p
    2. 생각 뒤집기를 통해 아이디어를 생각해 보아라. 291p
    3. 아이디어의 부분수정은 생각랜드가 아닌 나만의 데이터에 기초해야한다.
8장 완성사례: 버스U
1. 단계별 전환율 : 관심을 보인 사람들 중 아주 소수의 사람만이 유료 고객으로 전환됨. 304p
2. 적극적 투자 의 정도에 따라 xyz 가설의 x 값도 달라짐.
    1. 아이디어를 검증하는 과정에서의 적극적 투자가 포함된 나만의 데이터를 수집했다. 이는 투자를 받으려는 경우 강력한 사업근거가 된다.328p
    2. 버스 U는 해피엔딩을 가리키지만 처음 생각한 아이디어1 (3000달러의 10주 짜리 수업)은 안될놈이였다. 테스트와 수정을 통해 될놈버전이 된것이다.
    3. 구체적인 사건의 전개나 결과에 따라 배운 도구와 전략을 결합하고 펼치는 것이 달라질 것이다.
9장 마지막 당부
1.  그 아이디어가 ‘우리를 위한’ ‘될놈’ 인가 ? 353p
    1. ‘될놈’을 발견하게 되면 제대로 대접할수 있게 준비해라. 성공 역시 야수가 될 수 있다.
    2. 내가 ‘될 놈’ 에게 흥미를 가지고 있는가?
2. 세상을 위한 ‘될 놈’인가? 359p
        1. 한 몫 챙기는 것 이상을 생각하라. 쓸모가 전혀 없는 것들은 만들지마라
        2. 비즈니스 너머를 생각하라. 아직도 세상의 심각한 문제들은 이것을 해결해줄 ‘될 놈’을 찾고 있다.
3. 올바른 ‘될 놈’을 추구하라
    1. 우리가 정말로 관심 갖는 문제를 해결하고 관심 갖는 시장에 이바지할려고 할 때 어떤 난관이 닥쳐도 굿굿이 해나가면서 극복할 수 있는 에너지와 동기가 생길 것이다.
    2. 우리의 제품이 세상에 가치를 가지고 도움이 되면 저항이 휠씬 적을 뿐만 아니라 예상치 못한 온갖 사람과 단체가 나타나 우리를 응원해주고 도와줄 것이다. 364p

제가 요약한 아이디어 불패의 법칙입니다 !

**Reply by hanseulhee** [reply]
왓 완전 정리 잘하셨네요 👍🏻👍🏻👍🏻
